src/azul/indexer/index_controller.py

```diff
@@ -274,6 +292,20 @@ def aggregate(self, event: Iterable[SQSRecord], *, retry=False):
             log.warning('Failed to aggregate tallies: %r', tallies_by_entity.values(), exc_info=True)
             raise
 
+    def redrive(self, event):
+        for record in event:
+            message = json.loads(record.body)
+            receive_count = record.to_dict()['attributes']['ApproximateReceiveCount']
```

Comment:
As above, consider converting to `int` on this line to facilitate reuse.

Reply:
@achave11 You gave this comment a :+1: but didn't implement the suggestion, was that intentional?

Reply:
Not intentional. I addressed one of the two occurrences, which is why I 👍🏽, but I should have addressed both. Thank you for catching that.